### PR TITLE
tidb-in-kubernetes/get-started: add deploy-tidb-from-kubernetes-kind.md

### PR DESCRIPTION
**Pull request by anotherrachel**
### What is changed, added or deleted? 
This PR adds a doc to introduce how to deploy a TiDB cluster in Kubernetes using kind.
**The `TOC.md` is not updated accordingly.**

### What is the related PR or file link(s)? 
https://github.com/pingcap/docs-cn/pull/1855

### Which version does your change affect? 
dev, v3.0

### CODE REVIEW

**Comment by anotherrachel** [comment]
@Yisaer @yikeke PTAL, thanks~

**Comment by lilin90** [comment]
@yikeke PTAL

File: dev/tidb-in-kubernetes/get-started/deplpy-tidb-from-kubernetes-kind.md [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+
+    {{< copyable "shell-regular" >}}
+
+    ``` shell
+    hack/kind-cluster-build.sh
+    ```
+
+    > **Note:**
+    >
+    > In this script, the Kubernetes version defaults to v1.12.8, and the Kubernetes cluster starts with six nodes and for each node the mount count set to 9. You can configure these items in startup options.
```
Comment (yikeke) [inline dev/tidb-in-kubernetes/get-started/deplpy-tidb-from-kubernetes-kind.md]:
```suggestion
    > By default, this script starts a Kubernetes cluster of the 1.12.8 version, with six nodes in the cluster and for each node the number of mount points is 9. You can configure these items by startup options.
```

File: dev/tidb-in-kubernetes/get-started/deplpy-tidb-from-kubernetes-kind.md [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+    >
+    > {{< copyable "shell-regular" >}}
+    >
+    > ```shell
+    > hack/kind-cluster-build.sh --nodeNum 2 --k8sVersion v1.14.6 --volumeNum 3
+    > ```
+
+3. Execute the following command to set the default configuration files of kubectl to `kube-config`, so as to connect the local Kubernetes cluster.
```
Comment (yikeke) [inline dev/tidb-in-kubernetes/get-started/deplpy-tidb-from-kubernetes-kind.md]:
```suggestion
3. To connect the local Kubernetes cluster, set the default configuration file path of kubectl to `kube-config`.
```

File: dev/tidb-in-kubernetes/get-started/deplpy-tidb-from-kubernetes-kind.md [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    export KUBECONFIG="$(kind get kubeconfig-path)"
+    ```
+
+4. You can verify whether the Kubernetes cluster is on and running by using the following command:
```
Comment (yikeke) [inline dev/tidb-in-kubernetes/get-started/deplpy-tidb-from-kubernetes-kind.md]:
```suggestion
4. Verify whether the Kubernetes cluster is on and running:
```

File: dev/tidb-in-kubernetes/get-started/deplpy-tidb-from-kubernetes-kind.md [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+
+    {{< copyable "shell-regular" >}}
+
+    ``` shell
+    kubectl cluster-info
+    ```
+
+    The response should be like this:
```
Comment (yikeke) [inline dev/tidb-in-kubernetes/get-started/deplpy-tidb-from-kubernetes-kind.md]:
```suggestion
    The output is like this:
```

File: dev/tidb-in-kubernetes/get-started/deplpy-tidb-from-kubernetes-kind.md [hunk 5]
```diff
@@ -0,0 +1,132 @@
...
+
+5. Check the `storageClass` of the cluster:
+
+    {{< copyable "shell-regular" >}}
+
+    ``` shell
+    kubectl get storageClass
+    ```
+
+    The response should be like this:
```
Comment (yikeke) [inline dev/tidb-in-kubernetes/get-started/deplpy-tidb-from-kubernetes-kind.md]:
```suggestion
    The output is like this:
```

File: dev/tidb-in-kubernetes/get-started/deplpy-tidb-from-kubernetes-kind.md [hunk 6]
```diff
@@ -0,0 +1,132 @@
...
+
+    ``` shell
+    NAME                 PROVISIONER                    AGE
+    local-storage        kubernetes.io/no-provisioner   7m50s
+    standard (default)   kubernetes.io/host-path        8m29s
+    ```
+
+## Step 2: Install TiDB Operator in the Kubernetes
```
Comment (yikeke) [inline dev/tidb-in-kubernetes/get-started/deplpy-tidb-from-kubernetes-kind.md]:
```suggestion
## Step 2: Deploy TiDB Operator in the Kubernetes cluster
```

File: dev/tidb-in-kubernetes/get-started/deplpy-tidb-from-kubernetes-kind.md [hunk 7]
```diff
@@ -0,0 +1,132 @@
...
+
+Refer steps in [Deploy TiDB Operator](/dev/tidb-in-kubernetes/deploy/tidb-operator.md#install-tidb-operator).
```
Comment (yikeke) [inline dev/tidb-in-kubernetes/get-started/deplpy-tidb-from-kubernetes-kind.md]:
```suggestion
Refer to the steps in [Deploy TiDB Operator](/dev/tidb-in-kubernetes/deploy/tidb-operator.md#install-tidb-operator).
```

File: dev/tidb-in-kubernetes/get-started/deplpy-tidb-from-kubernetes-kind.md [hunk 8]
```diff
@@ -0,0 +1,132 @@
...
+
+## Step 3: Deploy a TiDB cluster in the Kubernetes cluster
+
+Refer steps in [Deploy TiDB on General Kubernetes](/dev/tidb-in-kubernetes/deploy/general-kubernetes.md#deploy-tidb-cluster).
```
Comment (yikeke) [inline dev/tidb-in-kubernetes/get-started/deplpy-tidb-from-kubernetes-kind.md]:
```suggestion
Refer to the steps in [Deploy TiDB on General Kubernetes](/dev/tidb-in-kubernetes/deploy/general-kubernetes.md#deploy-tidb-cluster).
```

File: dev/tidb-in-kubernetes/get-started/deplpy-tidb-from-kubernetes-kind.md [hunk 9]
```diff
@@ -0,0 +1,132 @@
...
+
+## Access the database and monitor dashboards
```
Comment (yikeke) [inline dev/tidb-in-kubernetes/get-started/deplpy-tidb-from-kubernetes-kind.md]:
```suggestion
## Access the database and monitoring dashboards
```

File: dev/tidb-in-kubernetes/get-started/deplpy-tidb-from-kubernetes-kind.md [hunk 10]
```diff
@@ -0,0 +1,132 @@
...
+
+Refer the steps in [View the monitoring dashboard](/dev/tidb-in-kubernetes/monitor/tidb-in-kubernetes.md#view-the-monitoring-dashboard).
+
+## Destroy the TiDB and Kubernetes cluster
+
+To destroy the local TiDB cluster, refer the steps in [Destroy TiDB Clusters in Kubernetes](/dev/tidb-in-kubernetes/maintain/destroy-tidb-cluster.md).
```
Comment (yikeke) [inline dev/tidb-in-kubernetes/get-started/deplpy-tidb-from-kubernetes-kind.md]:
```suggestion
To destroy the local TiDB cluster, refer to the steps in [Destroy TiDB Clusters in Kubernetes](/dev/tidb-in-kubernetes/maintain/destroy-tidb-cluster.md).
```

File: dev/tidb-in-kubernetes/get-started/deplpy-tidb-from-kubernetes-kind.md [hunk 11]
```diff
@@ -0,0 +1,132 @@
...
+
+Refer steps in [Deploy TiDB Operator](/dev/tidb-in-kubernetes/deploy/tidb-operator.md#install-tidb-operator).
+
+## Step 3: Deploy a TiDB cluster in the Kubernetes cluster
+
+Refer steps in [Deploy TiDB on General Kubernetes](/dev/tidb-in-kubernetes/deploy/general-kubernetes.md#deploy-tidb-cluster).
+
+## Access the database and monitor dashboards
+
+Refer the steps in [View the monitoring dashboard](/dev/tidb-in-kubernetes/monitor/tidb-in-kubernetes.md#view-the-monitoring-dashboard).
```
Comment (yikeke) [inline dev/tidb-in-kubernetes/get-started/deplpy-tidb-from-kubernetes-kind.md]:
```suggestion
Refer to the steps in [View the monitoring dashboard](/dev/tidb-in-kubernetes/monitor/tidb-in-kubernetes.md#view-the-monitoring-dashboard).
```

**Comment by anotherrachel** [comment]
@yikeke @juliezhang1112 All comments addressed, PTAL again~ 
When changes are approved, I'll update `TOC.md` and v3.0 soon.

File: dev/tidb-in-kubernetes/get-started/deplpy-tidb-from-kubernetes-kind.md [hunk 12]
```diff
@@ -0,0 +1,132 @@
...
+title: Deploy TiDB in Kubernetes Using kind
+summary: Learn how to deploy a TiDB cluster in Kubernetes using kind.
+category: how-to
+---
+
+# Deploy TiDB in Kubernetes Using kind
+
+This tutorial shows how to deploy [TiDB Operator](https://github.com/pingcap/tidb-operator) and a TiDB cluster in Kubernetes on your laptop (Linux or macOS) using [kind](https://kind.sigs.k8s.io/).
+
+kind is a tool for running local Kubernetes clusters using Docker containers as cluster nodes. It is developed for testing local Kubernetes clusters, initially targeting the conformance tests. The cluster version depends on the node image that your kind version uses, but you can specify the image to be used for the nodes and choose any other published version. Refer to [Docker hub](https://hub.docker.com/r/kindest/node/tags) to see available tags.
```
Comment (yikeke) [inline dev/tidb-in-kubernetes/get-started/deplpy-tidb-from-kubernetes-kind.md]:
```suggestion
kind is a tool for running local Kubernetes clusters using Docker containers as cluster nodes. It is developed for testing local Kubernetes clusters, initially targeting the conformance tests. The Kubernetes cluster version depends on the node image that your kind uses, and you can specify the image to be used for the nodes and choose any other published version. Refer to [Docker hub](https://hub.docker.com/r/kindest/node/tags) to see available tags.
```

File: dev/tidb-in-kubernetes/get-started/deploy-tidb-from-kubernetes-kind.md [hunk 13]
```diff
@@ -0,0 +1,132 @@
...
+title: Deploy TiDB in Kubernetes Using kind
+summary: Learn how to deploy a TiDB cluster in Kubernetes using kind.
+category: how-to
+---
+
+# Deploy TiDB in Kubernetes Using kind
+
+This tutorial shows how to deploy [TiDB Operator](https://github.com/pingcap/tidb-operator) and a TiDB cluster in Kubernetes on your laptop (Linux or macOS) using [kind](https://kind.sigs.k8s.io/).
+
+kind is a tool for running local Kubernetes clusters using Docker containers as cluster nodes. It is developed for testing local Kubernetes clusters, initially targeting the conformance tests. The Kubernetes cluster version depends on the node image that your kind uses, and you can specify the image to be used for the nodes and choose any other published version. Refer to [Docker hub](https://hub.docker.com/r/kindest/node/tags) to see available tags.
```
Comment (juliezhang1112) [inline dev/tidb-in-kubernetes/get-started/deploy-tidb-from-kubernetes-kind.md]:
```suggestion
kind is a tool for running local Kubernetes clusters using Docker containers as cluster nodes. It is developed for testing local Kubernetes clusters, initially targeting the conformance tests. The Kubernetes cluster version depends on the node image that your kind uses, and you can specify the image to be used for the nodes and choose any published version. Refer to [Docker hub](https://hub.docker.com/r/kindest/node/tags) to see available tags.
```
@yikeke  If we use "and" to join the sentence, `other` in `any other` should be removed here.
FYI, https://stackoverflow.com/questions/56046451/use-kind-to-install-different-kubernetes-version

**Pull request merged**